app/dashboard/views.py

```diff
@@ -2914,6 +2927,127 @@ def get_profile_tab(request, profile, tab, prev_context):
         raise Http404
     return context
 
+def get_profile_by_idena_token(token):
+    try:
+        return Profile.objects.get(idena_token=token)
+    except ObjectDoesNotExist:
+        return None
+
+@require_POST
+def logout_idena(request, handle):
+    is_logged_in_user = request.user.is_authenticated and request.user.username.lower() == handle.lower()
+   
+    if not is_logged_in_user:
+        return JsonResponse({
+            'ok': False,
```

Comment:
Lets be consistent in our error response here - here we have 'ok' and 'msg' keys, below we have 'success' and 'error' keys.

Reply:
I did that intentionally because this API will call by Idena client and it excepts the response be like `success` and `error`.
Will add a comment for that.